client/src/main/view/component/local/imagens-local.html

```diff
@@ -13,12 +13,16 @@
         <!--end Label para o Input, o que é visto-->
     </div>
 
-    <!-- TODO: Cards com as imagens -->
     <!--Visualização das imagens-->
-    <div layout="row" layout-align="center center" layout-wrap>
-        <div ng-repeat="src in $ctrl.local.imagens">
-            <img ng-if="src.conteudo" ng-src="{{src.conteudo}}"/><br/>
-        </div>
+    <div flex="50" class="container-carousel" ng-if="$ctrl.existemImagens()">
+        <ui-carousel slides="$ctrl.local.imagens" slides-to-show="3" slides-to-scroll="1" dots="true" layout="row" layout-align="center center">
+            <carousel-item>
+                <md-card ng-click="$parent.$parent.$ctrl.mostraModalImagem($event, item, $parent.$parent.$ctrl.editavel)">
+                    <img ng-src="{{item.conteudo}}" class="imagem-local-card"/>
```

Comment:
por que precisa usar o $parent.$parent ? Se for por nome, nomeia o controller e usa ele diretamente aqui, o de imagemLocal